lib/ello_click/affiliate/threadless.ex

```diff
@@ -0,0 +1,34 @@
+defmodule ElloClick.Affiliate.Threadless do
+  alias ElloClick.Affiliate.Link
+
+  @ello_fake_threadless_affiliate "xrGWivQiB3WN3lVUxIzErxGGUkkz3ESwEVOtXE0"
+
+  # Don't re-affiliate links something further up the chain got
+  def affiliate(%Link{is_affiliated: true} = link, _conn), do: link
```

Comment:
@alanpeabody question for you - how are the pattern match options evaluated in Elixir? Is it order-of-declaration, or most-specific-to-least-specific, or something different?